src/App.js

```diff
@@ -59,38 +60,60 @@ function App({ data }) {
   return (
     <Wrapper>
       <Header />
-      <Switch>
-        <Route exact path="/">
-          {data.product.map(product => (
-            <ProductCard
-              description={product.description}
-              claim={product.claim}
-              price={product.price}
-              ingredients={product.ingredients}
-              alcohol={product.alcohol}
-              mount={product.mount}
-              image={product.image}
-              logo={product.logo}
-              key={product.name}
-            />
-          ))}
-          <Shop onAddOrder={handleAddOrder} />
-        </Route>
-        <Route exact path="/motive">
-          {data.campaign.map(campaign => (
-            <CampaignCard image={campaign.image} />
-          ))}
-        </Route>
-      </Switch>
+      <Main>
+        <Switch>
+          <Route exact path="/">
+            {data.product.map(product => (
+              <ProductCard
+                description={product.description}
+                claim={product.claim}
+                price={product.price}
+                ingredients={product.ingredients}
+                alcohol={product.alcohol}
+                mount={product.mount}
+                image={product.image}
+                logo={product.logo}
+                key={product.name}
+              />
+            ))}
+            <Shop onAddOrder={handleAddOrder} />
+          </Route>
+          <Route exact path="/motive">
+            {data.campaign.map(campaign => (
+              <CampaignCard image={campaign.image} />
```

Comment:
There is a `key` missing here.
```suggestion
              <CampaignCard image={campaign.image} key={campaign.image} />
```